### PR TITLE
Issue #30: Specify size for screenshots, remove fixed size

### PR DESCRIPTION
**Pull request by xurizaemon**

### CODE REVIEW

**Comment by AlexSkrypnyk** [comment]
@xurizaemon Thank you so much for contributing.

Could you please add a simple test similar to https://github.com/integratedexperts/behat-screenshot/blob/master/tests/behat/features/selenium.feature - just to call this step definition to make sure that it works.

Thanks

File: src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php [hunk 1]
```diff
@@ -160,6 +159,21 @@ public function iSaveScreenshot($fail = false)
...
+     *
+     * @param int $width  Width to resize browser to.
+     * @param int $height Height to resize browser to.
+     *
+     * @When save :width x :height screenshot
+     * @When I save :width x :height screenshot
+     */
+    public function iSaveSizedScreenshot($width = 1440, $height = 900)
+    {
+        $this->getSession()->resizeWindow((int) $width, (int) $height, 'current');
```
Comment (AlexSkrypnyk) [inline src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php:173]: @xurizaemon 
Could you please put a check if the current session support window resize. With how this code currently written, using this step in non-JS enabled feature would trigger an exception.

Similar to https://github.com/integratedexperts/behat-screenshot/blob/master/src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php#L134

**Review by AlexSkrypnyk — CHANGES_REQUESTED** [review]

File: src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php (hunk 1 above)
Comment (xurizaemon) [reply src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php]: Done. (Is there a better method to check driver capabilities than catching UnsupportedDriverActionException as I've done here?)

File: src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php [hunk 2]
```diff
@@ -86,7 +86,6 @@ public function beforeScenarioInit(BeforeScenarioScope $scope)
                 if (!$driver->isStarted()) {
                     $driver->start();
                 }
-                $this->getSession()->resizeWindow(1440, 900, 'current');
```
Comment (AlexSkrypnyk) [inline src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php:89]: @xurizaemon 
Can we please get this reverted? I'm not sure what is the reason to remove this.

This was set as one of the most widely-used full-screen sizes.

The added step definition will override the value set in before scenario.

Comment (xurizaemon) [reply src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php:89]: Sure! (This was WIP towards configuring the default screen size from behat.yml, but that can wait.)

Comment (xurizaemon) [reply src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php]: Done :)

**Comment by AlexSkrypnyk** [comment]
@xurizaemon thank you for your contribution!

**Pull request merged**